proxy/tcpproxy/userspace.go

```diff
@@ -33,16 +39,18 @@ func (r *remote) inactivate() {
 	r.inactive = true
 }
 
-func (r *remote) tryReactivate() {
+func (r *remote) tryReactivate() error {
```

Comment:
please change this back. the blocking point is net.Dial. we need to ensure the whole func is unblocking.